.github/workflows/build_and_test_feature.yml

```diff
@@ -248,7 +254,7 @@ jobs:
 
         - name: Run visual regression tests
           run: |
-            cd packages/icon-library/ && tar -xzf ../../dist.tar.gz && cd ../..
+            tar -xzf dist.tar.gz && mv dist packages/icon-library/ && mv distdt packages/design-tokens/dist
```

Comment:
Maybe we should call the icon-library dist folder something else while it's being moved about?

`icon-library_dist.tar.gz` & `design-tokens_dist.tar.gz`?

Reply:
icon library directory renamed to 'distil' whilst it is in transit